nav2_system_tests/src/planning/test_planner_node.cpp

```diff
@@ -37,27 +38,56 @@ class RclCppFixture
 
 RclCppFixture g_rclcppfixture;
 
-TEST_F(PlannerTester, testSimpleCostmaps)
-{
-  std::vector<TestCostmap> costmaps = {
-    TestCostmap::open_space,
-    TestCostmap::bounded,
-    TestCostmap::top_left_obstacle,
-    TestCostmap::bottom_left_obstacle,
-    TestCostmap::maze1,
-    TestCostmap::maze2
-  };
-
-  ComputePathToPoseResult result;
-
-  for (auto costmap : costmaps) {
-    loadSimpleCostmap(costmap);
-    EXPECT_EQ(true, defaultPlannerTest(result));
-  }
-}
+// #TODO(orduno) This test throws a fastrtps error most of the time when bringing down navfn
```

Comment:
Do you plan to re-enable these tests in the near future? If so, let's file an issue so it doesn't get forgotten. If not, let's just delete this code instead of commenting it out.

Reply:
Removing, the following test will capture starting and shutting down the `navnfn` node.

Reply:
However, the following test will remain commented until #1100 is resolved.